src/pe_reports/report_generator.py

```diff
@@ -51,13 +54,13 @@ def export_set(prs, out_dir):
         logging.error("%s : Missing input data. No report generated.", not_found)
 
 
-def generate_reports(data, data_dir, out_dir):
+def generate_reports(data, data_dir, out_dir, db_creds_file):
     """Gather assets to produce reports."""
     # TODO: build code to connect customer db, encrypt and embed pdf reports.
     # Issue 7: https://github.com/cisagov/pe-reports/issues/7
 
 
-def main() -> None:
+def main():
```

Comment:
Not sure why this change was made, but it should be reverted:
```suggestion
def main() -> None:
```